demisto_sdk/commands/common/content/objects/pack_objects/pre_preocess_rules/pre_process_rules.py

```diff
@@ -0,0 +1,45 @@
+from typing import Union
+
+import demisto_client
+from wcmatch.pathlib import Path
+
+from demisto_sdk.commands.common.constants import PRE_PROCESS_RULES, FileType
+from demisto_sdk.commands.common.content.objects.pack_objects.abstract_pack_objects.json_content_object import \
+    JSONContentObject
+from demisto_sdk.commands.format.update_pre_process_rules import \
+    PRE_PROCESS_RULES_PREFIX
+
+
+class PreProcessRules(JSONContentObject):
```

Comment:
should be singular, each file/object is a single rule
```suggestion
class PreProcessRule(JSONContentObject):
```

Reply:
https://github.com/demisto/demisto-sdk/commit/cf2dab1ba20b8675faeeadb856217b7e99d105b0